datanode/tools/calculate_datanode_metrics.bash

```diff
@@ -1,7 +1,7 @@
 #/bin/bash
```

Comment:
This script doesn't work in a Docker config (nor would it work with gunicorn), correct? Let's get rid of it and we can have a separate issue for adding the metrics we need, even in a Docker config.

Reply:
Deleted.